README.md

```diff
@@ -1,18 +1,31 @@
 # vscode-flowr
-flowR Extension for Visual Studio Code 
+*flowR* Extension for Visual Studio Code. For more information on *flowR*, check out [the main repository](https://github.com/Code-Inspect/flowr).
 
-## Features
-TO DO
+**This extension is still work in progress, as is this README. Please stay tuned for cool features and cooler documentation!**
 
-## Requirements
-TO DO
+## Installing
 
-## Extension Settings
-TO DO
+### From Visual Studio Marketplace
+vscode-flowr is not available on the Visual Studio Marketplace yet. For now, we recommend installing from the artifact, or building from source.
 
-# Developing
+### From GitHub Releases
+There are no official releases yet. Stay tuned!
 
-## Git Hooks
+### From build artifacts
+You can easily download the most recent build of the extension by heading to the [Actions tab](https://github.com/Code-Inspect/vscode-flowr/actions/workflows/package.yml), where you will find a list of runs. Selecting the most recent run will display a summary of it, at the bottom of which you can find the Artifacts section and the `Extension vsix` artifact. Download it und unzip it.
+
+From Visual Studio Code, open the Extensions tab and click on the three dots in the top right to select "Install from VSIX..." Alternatively, you can use the Command Palette to select the option directly. Then, you can select the `vscode-flowr-[version].vsix` file you unzipped to install it.
+
```

Comment:
Can you please add a section showing how to use the extension with a simple example? (e.g., open R file, click variable, ...)

Furthermore, maybe we should add a link to vscode-R as a "recommendation".